llvm/test/Transforms/LoopVectorize/reduction-inloop.ll

```diff
@@ -1,5 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
 ; RUN: opt < %s -passes=loop-vectorize,dce,instcombine -force-vector-interleave=1 -force-vector-width=4 -prefer-inloop-reductions -S | FileCheck %s
+; Check remark analysis runs successfully.
+; RUN: opt < %s -passes=loop-vectorize,dce,instcombine -force-vector-interleave=1 -force-vector-width=4 -prefer-inloop-reductions -pass-remarks-analysis=loop-vectorize -S
```

Comment:
Might be better to have a separate test case that also checks the output of the remark as well? Also no need to run `dce,instcombine`.

Reply:
Add new test case for this failure, thanks.